### ISSUE TITLE
trouble with breaking blocks

### ISSUE THREAD

**Issue by HuffleQueen** [jaquadro/StorageDrawers#873]
hi so in creative mode the block are not breaking as they should and some also refuse to break or take way to long like others blocks would in this game mode, it doesn't work with using tools either and i tried in survival as well and it works find in survival, just in creative ( sorry if this seems all over the place, not good at explaining)

**Comment by rlane85** [comment]
I am having the same issue. Currently using the Create Stellar modpack in creative mode. I tried breaking my drawers in creative mode normally, I tried breaking while using a combination of several keys (shift, control, alt, etc.), and I tried breaking while crouching. I also read in the wiki that clicking on the "front" of the drawer is how you interact with it, so I made sure to try breaking from the back. But nothing works except switching to survival (came here to find this, so thanks!). 

TL;DR Can't break drawers in creative, only in survival

MC 1.19.2
StorageDrawers-1.19-11.1.2.jar
Forge 43.2.8
Create Stellar-v1.1.8
